CONTRIBUTING.md

```diff
@@ -12,19 +12,23 @@ Machine Translate is **created and edited by [contributors like you!](contributi
 
 We recommend reading the [**roadmap**](/roadmap.md) and [**style guide**](contributing/style.md) before you start contributing.
 
-
 ### Creating and editing articles
 
 To create or edit an article, [**create**](https://github.com/machinetranslate/machinetranslate.org/new/master) or [**edit**](https://github.com/machinetranslate/machinetranslate.org) and send us a [pull request](https://github.com/machinetranslate/machinetranslate.org/pulls?q=is%3Apr).
 
-> See GitHub’s article [*Creating a pull request*](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request#creating-the-pull-request) for more
+> See GitHub’s article [*Creating a pull request*](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request#creating-the-pull-request) for more.
+
+### Reviewing articles
```

Comment:
Maybe it should be

> ### Reviewing creation and edits

or just 

> ### Reviewing edits

or 

> ### Reviewing new and edited articles

Not sure.

Reply:
Changed to `reviewing proposed changes`.